apps/evm/src/components/Table/TableCards.tsx

```diff
@@ -122,7 +109,11 @@ export function TableCards<R>({
           return (
             <Card
               key={rowKey}
-              css={styles.tableWrapperMobile({ clickable: !!(rowOnClick || getRowHref) })}
+              className={cn(
+                '[&:not(:last-of-type)]:mb-4',
```

Comment:
Tailwind has a built-in class for this: `space-y-4`, which you can add to the parent to apply a gap of 4 between each child.